src/App.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import { Route } from "react-router-dom"; 
 
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path={'/'} component={Home}/>
```

Comment:
The curly brackets work but not necessary. They would be useful however if you want to use a template literal.
```suggestion
    <Route exact path='/' component={Home}/>
```